.github/workflows/embedding-sdk.yml

```diff
@@ -186,3 +186,10 @@ jobs:
               }
             });
             process.exit(1);
+
+  cross-version-for-breaking-changes:
+    uses: ./.github/workflows/e2e-cross-version-for-breaking-changes.yml
+    secrets: inherit
+    with:
+      fe-ref: ${{ github.base_ref }}
+      be-ref: ${{ github.head_ref }}
```

Comment:
Should it be the other way around? As far as I know `base_ref` is target (e.g. `master`) and `head_ref` is our branch. Or is this intended for backend folks when they're working on their backend PRs?

Reply:
I think I agree with Nicolo, we should use new backend with an old frontend to catch if a backend code change is  breaking compatibility with the existing frontend. I think this also allows this to pass if we've added non-breaking new features (e.g. new viz type) because we're running the old tests against the old frontend, and the new backend can just sit there unexercised.